NDTensors/src/lib/Sectors/src/abstractcategory.jl

```diff
@@ -0,0 +1,41 @@
+abstract type AbstractCategory end
+
+abstract type AbstractGroup <: AbstractCategory end
```

Comment:
The `AbstractGroup` doesn't seem very useful, it doesn't look like there are any functions overloaded for `AbstractGroup`.

Reply:
Agreed, I put it in there mostly as a discussion point. While I can envision that some things may only be defined for groups, and not for categories, my sense is that these finer differences can just be kind of 'emergent' and more of like a 'duck typing' thing for each category.

Reply:
I would vote for removing it for now just to keep the code simpler, and adding it back if needed.